test/test_profiler.py

```diff
@@ -1317,9 +1317,38 @@ def test_profiler_pattern_match_helper(self):
         self.assertEqual(event_tree[1], pattern.next_of(event_tree[0]))
         self.assertEqual(event_tree[0], pattern.prev_of(event_tree[1]))
 
-
-
-
+    @unittest.skipIf(not torch.cuda.is_available(), "CUDA is required")
+    def test_profiler_extra_cuda_copy_pattern(self):
+        from torch.profiler._pattern_matcher import ExtraCUDACopyPattern
+
+        with profile(activities=[ProfilerActivity.CPU, ProfilerActivity.CUDA],
```

Comment:
nit: do we need `activities=[ProfilerActivity.CPU, ProfilerActivity.CUDA]` on all of them? (It's just cluttering the screen a bit)